ui/app/components/namespace-picker.js

```diff
@@ -157,4 +157,10 @@ export default Component.extend({
     let parts = namespace.split('/');
     return parts[parts.length - 1];
   }),
+
+  actions: {
+    refreshNamespaceList() {
+      this.get('namespaceService.findNamespacesForUser').perform();
```

Comment:
Do we need to provide any params here to make sure the result we fetch isn't cached?

Reply:
Good question.  I see `.perform()` used throughout the codebase without any params so I am guessing no, we're just calling the method.